governance.md

```diff
@@ -0,0 +1,320 @@
+# Main Governance Document
+
+The official version of this document, along with a list of
+individuals and institutions in the roles defined in the governance
+section below.
+
+
+The Project
+===========
+
+The PyMC3 Project (The Project) is an open source software project
+affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to
+develop open source software and deploy open and public websites and services
+for reproducible, exploratory and interactive computing. The Software developed
+by The Project is released under the Apache 2 open source license,
+developed openly and hosted in public GitHub repositories under the
+[GitHub organization](https://github.com/pymc-devs/pymc3). Examples of
+Project Software include the PyMC3 code and the Documentation, etc. The Services run by the
+Project consist of public websites and web-services that are hosted
+under at ([http://pymc-devs.github.io/pymc3/](http://pymc-devs.github.io/pymc3/) 
+
+The Project is developed by a team of distributed developers, called
+Contributors. Contributors are individuals who have contributed code,
+documentation, designs or other work to one or more Project repositories.
+Anyone can be a Contributor. Contributors can be affiliated with any legal
+entity or none. Contributors participate in the project by submitting,
+reviewing and discussing GitHub Pull Requests and Issues and participating in
+open and public Project discussions on GitHub, Google+, Slack, Hackpad, Gitter chat
+rooms and mailing lists. The foundation of Project participation is openness
+and transparency.
+
+Here is a list of the current Contributors to the main  repository:
+John Salvatier
+Chris Fonnesbeck
+Thomas Wiecki
+Peadar Coyle
+Taku Yoshioka
+Austin Rochford
+Jon Sedar
+
+
+There are also many other Contributors listed in the logs of other repositories of
+the PyMC3 and  projects.
```

Comment:
Need to fix this and rebase a bit.
And do those fixes
On 11 Jul 2016 10:55 PM, "Chris Fonnesbeck" <notifications@github.com>
wrote:

> In governance.md
> <https://github.com/pymc-devs/pymc3/pull/1217#discussion_r70344736>:
>
> > +open and public Project discussions on GitHub, Google+, Slack, Hackpad, Gitter chat
> > +rooms and mailing lists. The foundation of Project participation is openness
> > +and transparency.
> > +
> > +Here is a list of the current Contributors to the main  repository:
> > +John Salvatier
> > +Chris Fonnesbeck
> > +Thomas Wiecki
> > +Peadar Coyle
> > +Taku Yoshioka
> > +Austin Rochford
> > +Jon Sedar
> > +
> > +
> > +There are also many other Contributors listed in the logs of other repositories of
> > +the PyMC3 and  projects.
>
> of PyMC and related projects
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pymc-devs/pymc3/pull/1217/files/11ff221c0ddbd58ae451a403e76cb5d0cd498f50#r70344736>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA8DiER8jRu2-BgxUOCAbHKOnc4rtUa0ks5qUru4gaJpZM4JCsyk>
> .
>